src/main/java/gov/usgs/cida/pubs/webservice/mp/MpListMvcService.java

```diff
@@ -32,7 +30,7 @@ public MpListMvcService(@Qualifier("mpListBusService") final IBusService<MpList>
 		this.busService = busService;
 	}
 
-	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
+//	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
```

Comment:
and here, I'll stop commenting but there are others as well that you may or may not want to delete.